templates/projects/task_presenter_editor.html

```diff
@@ -143,6 +143,12 @@
                                 <td><p>Last Commit:</p></td>
                                 <td class="ref-details"><a href={{ guidelines_github.ref_url }} target="_blank">{{ guidelines_github.ref }}</a></td>
                             </tr>
+                            {% if guidelines_github.error %}
```

Comment:
As guidelines_github_exists can be set to true based on multiple factors that may not be set when there's error, can this be outside of {% if guidelines_github_exists %} block and be instead
> {% if guidelines_github and guidelines_github.error %}

Same for task_presenter_github_exists

Reply:
How about this so I can fit it all in one table?

Reply:
technically yes, however meaning of guidelines_github_exists changes to guidelines_github_exists_or_error

Reply:
Now it's perfect.